### ISSUE TITLE
Dandelion Optimizer: A nature-inspired metaheuristic algorithm for engineering applications

### ISSUE THREAD

**Issue by dpcuervo** [fcampelo/EC-Bestiary#158]
https://doi.org/10.1016/j.engappai.2022.105075

This paper proposes a novel [swarm intelligence](https://www.sciencedirect.com/topics/engineering/swarm-intelligence) bioinspired optimization algorithm, called the Dandelion Optimizer (DO), for solving continuous [optimization problems](https://www.sciencedirect.com/topics/computer-science/optimization-problem). DO simulates the process of dandelion seed long-distance flight relying on wind, which is divided into three stages. In the rising stage, seeds raise in a spiral manner due to the eddies from above or drift locally in communities according to different weather conditions. In the descending stage, flying seeds steadily descend by constantly adjusting their direction in global space. In the landing stage, seeds land in randomly selected positions so that they grow. The moving trajectory of a seed in the descending stage and landing stage are described by Brownian motion and a Levy random walk.

**Comment by mlhs66** [comment]
Hello, can you provide the source code of the dandelion algorithm?

**Comment by fcampelo** [comment]
> Hello, can you provide the source code of the dandelion algorithm?

No. The Bestiary is not a code base, and we do not endorse or recommend the use of any of the methods listed. In fact we find most if not all of them quite ridiculous and probably useless, but it's up to each one to make up their own minds about these things.
